src/main/java/net/earthcomputer/clientcommands/command/arguments/MojmapPacketClassArgumentType.java

```diff
@@ -0,0 +1,68 @@
+package net.earthcomputer.clientcommands.command.arguments;
+
+import com.mojang.brigadier.StringReader;
+import com.mojang.brigadier.arguments.ArgumentType;
+import com.mojang.brigadier.context.CommandContext;
+import com.mojang.brigadier.exceptions.CommandSyntaxException;
+import com.mojang.brigadier.suggestion.Suggestions;
+import com.mojang.brigadier.suggestion.SuggestionsBuilder;
+import net.earthcomputer.clientcommands.features.MappingsHelper;
+import net.fabricmc.fabric.api.client.command.v2.FabricClientCommandSource;
+import net.fabricmc.mappingio.tree.MappingTreeView;
+import net.minecraft.commands.SharedSuggestionProvider;
+import net.minecraft.network.protocol.Packet;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Optional;
+import java.util.Set;
+import java.util.concurrent.CompletableFuture;
+import java.util.stream.Collectors;
+
+public class MojmapPacketClassArgumentType implements ArgumentType<Class<Packet<?>>> {
+
+    private static final Collection<String> EXAMPLES = Arrays.asList("ClientboundPlayerChatPacket", "ClientboundSystemChatMessage", "ServerboundContainerSlotStateChangedPacket");
+
+    public static final String MOJMAP_PACKET_PREFIX = "net/minecraft/network/protocol/game/";
+
+    private static final Set<String> mojmapPackets = MappingsHelper.mojmapClasses().stream()
```

Comment:
Instead of doing it this way, you can get a list of packet classes that are in use via the `ConnectionProtocol` enum. This way you are also not depending on Mojang's packet package prefix, which is good if they decide to change it.